### PR TITLE
Delete imapper

### PR DESCRIPTION
**Pull request by djino97**
Deleted IMapper<DbUser, string, object>

### CODE REVIEW

**Review by DarkFelas — APPROVED** [review]

File: src/UserService.Broker/Consumers/GetUserInfoConsumer.cs [hunk 1]
```diff
@@ -50,10 +47,17 @@ private object GetUserInfo(Guid userId)
...
 
-            return mapper.Map(dbUser, userPosition);
+            return new
+            {
+                UserId = dbUser.Id,
+                FirstName = dbUser.FirstName,
+                LastName = dbUser.LastName,
+                MiddleName = dbUser.MiddleName,
+                UserPosition = userPosition
+            };
```
Comment (lalagvana) [inline src/UserService.Broker/Consumers/GetUserInfoConsumer.cs:60]: Hm, is it okay to map right here? Maybe there's another way to rework it...

Comment (djino97) [reply src/UserService.Broker/Consumers/GetUserInfoConsumer.cs:60]: Do you have any suggestions? For example

Comment (djino97) [reply src/UserService.Broker/Consumers/GetUserInfoConsumer.cs:60]: Do you have any suggestions? For example

Comment (lalagvana) [reply src/UserService.Broker/Consumers/GetUserInfoConsumer.cs:60]: Maybe map it without UserPosition, and then set this field in the already existing DbUser

**Review by lalagvana — APPROVED** [review]

**Pull request merged**